user-service/src/controllers/UserCtrl.ts

```diff
@@ -116,4 +116,34 @@ export default class UserCtrl {
       return res.sendStatus(401).json({ message: "No user stored in cookie!" });
     }
   }
+
+  @Post("/delete")
+  async deleteUser(req: Request, res: Response) {
+    const { password, accessToken } = req.body;
+    if (!accessToken) {
+      return res.sendStatus(400).json({ message: "No jwt sent!" });
+    }
+    const verifiedToken = await verifyToken(accessToken);
+    if (!verifiedToken) {
+      return res.status(403).json({ message: "Invalid jwt!" });
+    }
```

Comment:
This works for now, maybe better to create a middleware like the example here - > https://tsed.io/docs/request-context.html#endpoint-metadata  in the future if there are more functions that require verifications.

Then other functions/classes can just use the middleware decorator making the code cleaner and SoC?